### ISSUE TITLE
make it work for objects other than ActiveModel

### ISSUE THREAD

**Issue by danpersa** [stouset/twitter_bootstrap_form_for#7]
For example if I use the session object 

<%= twitter_bootstrap_form_for(session, :url => sessions_path) do |f| %>

I get this error:

undefined method `model_name' for Rack::Session::Abstract::SessionHash:Class

**Comment by stouset** [comment]
You need to add an `:as => :session` for a normal `form_for` or `twitter_bootstrap_form_for` to work without ActiveModel objects. However it looks like I still expect there to be an ActiveModel-style `errors` attribute.

I'll patch it to make it optional.

**Comment by stouset** [comment]
This will go out in the next release. Let me know within the next day or so if you still have problems with non-ActiveModel objects. It's possible that I missed a corner case somewhere.

**Comment by danpersa** [comment]
ok, thanks for the fast reply!

**Comment by stouset** [comment]
The quickest way to test the patch to make sure it works for you is to replace the gem stanza in your Gemfile with:

``` ruby
gem 'twitter_bootstrap_form_for',
  :git => 'git://github.com/stouset/twitter_bootstrap_form_for.git'
```

**Comment by danpersa** [comment]
I verified with the gem generated from the repository, and it works for my use case.
I'll notify you if there are further problems related to this issue.

Thanks

**Comment by stouset** [comment]
I've got another fix to roll into 1.0.2, so this will go out later tonight or tomorrow morning.

**Comment by stouset** [comment]
Sorry, I completely forgot that this fix was pending a release. I've pushed out `1.0.2` with this change (and a few other improvements)
